test/widgets/action_sheet_test.dart

```diff
@@ -135,31 +168,42 @@ void main() {
       await tester.longPress(find.text(topic));
       // sheet appears onscreen; default duration of bottom-sheet enter animation
       await tester.pump(const Duration(milliseconds: 250));
-      check(find.byType(BottomSheet)).findsOne();
-      check(find.text('Follow topic')).findsOne();
-    });
+    }
+
+    Future<void> showFromAppBar(WidgetTester tester, {
+      ZulipStream? channel,
+      String topic = someTopic,
+      Iterable<StreamMessage>? messages,
```

Comment:
```suggestion
      List<StreamMessage>? messages,
```

Then we can skip the `toList` below. The callers are all passing lists anyway.